src/config.ts

```diff
@@ -68,7 +69,9 @@ const defaultConfig: ConfigInterface = {
   refreshWhenHidden: false,
   refreshWhenOffline: false,
   shouldRetryOnError: true,
-  suspense: false
+  suspense: false,
+
+  isEqualMethod: deepEqual
```

Comment:
```suggestion
  compare: deepEqual
```

I think it's better to call the option `compare` (same as React's `memo(component, compare)`).